model/project_ref.go

```diff
@@ -1475,11 +1475,31 @@ func (p *ProjectRef) UpdateAdminRoles(toAdd, toRemove []string) error {
 	return nil
 }
 
-func (p *ProjectRef) AuthorizedForGitTag(ctx context.Context, user string, token string) bool {
-	if utility.StringSliceContains(p.GitTagAuthorizedUsers, user) {
+func (p *ProjectRef) AuthorizedForGitTag(ctx context.Context, githubUser string, token string) bool {
+	if utility.StringSliceContains(p.GitTagAuthorizedUsers, githubUser) {
 		return true
 	}
-	return thirdparty.IsUserInGithubTeam(ctx, p.GitTagAuthorizedTeams, p.Owner, user, token)
+	// check if user has permissions with mana before asking github about the teams
+	u, err := user.FindByGithubName(githubUser)
+	if err != nil {
+		grip.Error(message.WrapError(err, message.Fields{
+			"message": "error checking if user is authorized for git tag",
+			"source":  "github hook",
```

Comment:
This seems to be a db check, and we don't seem to be accessing github until line 1502. Why do we want the source to be "github hook"?

Reply:
I'm using "github hook" to keep this error message linked to the logs that come from the Github webhook (which is the only time we use this function)

Reply:
makes sense, thanks!